networkit/cpp/sparsification/SimmelianScore.cpp

```diff
@@ -77,10 +77,10 @@ Redundancy SimmelianScore::getOverlap(	const node& ego,
  * egoNeighbors and alterNeighborsUnmatched to overlap.
  */
 void SimmelianScore::matchNeighbors(
-	const node& ego,
+	const node &,
```

Comment:
Passing a const ref to an integer/bool seems just wrong -- esp. when it's not used.

Reply:
Agreed.